vmdk_plugin/E2E_Tests/e2e_verifyVolmProperties_test.go

```diff
@@ -0,0 +1,133 @@
+        vmIp string = ""
+        dockerVolmRmvCmd string = "docker volume rm "
+        dockerCliCheck bool = true
+)
+
+func TestMain(m *testing.M) {
+	retCode := m.Run()
+	teardownFunction()
+	os.Exit(retCode)
+}
+
+// clean-up function
+func teardownFunction() {
+	TestUtil.ExecCmd(vmIp, dockerVolmRmvCmd)
+	log.Println("-----Clean-up finished - current time: ", time.Now())
+}
+
+/*
+Test steps:
+1. SSH to a vm and create a volume of size 100 mb and specified disk format ["thin", "zeroedthick", "eagerzeroedthick"]
+2. Do docker inspect on the volume and verify size, disk format and attached to vm field.
+   Expected value: {"100MB", "thin/zeroedthick/eagerzeroedthick", no-value} respectively.
+3. SSH to the esx and verify the capacity, disk format and attached-to-vm field for the volume using Admin cli
+   Expected value: {"100.00MB", "thin/zeroedthick/eagerzeroedthick", "detached"} respectively.
+4. SSH to the vm and run a container and mount the volume created
+5. SSH to the vm and esx and verify the attached-to-vm field for volume - both docker cli and admin cli values should be same.
+6. Again verify capacity and disk format using docker cli and admin cli to make sure things are fine after running the container.
+*/
+func TestVolumeProperties(t *testing.T) {
+	for vmIndx := 0; vmIndx < len(vms); vmIndx++ {
+		log.Println("running end_to_end tests - current time: ", time.Now())
+		vmIp = vms[vmIndx]
+
+		log.Println("Running test on VM - ", vmIp)
+		dockerCliCheck = TestUtil.IsDockerCliCheckNeeded(vms[vmIndx])
+		for i := 0; i < len(volSizes); i++ {
+			for k := 0; k < len(formatTypes); k++ {
+				log.Println("Creating a volume of Format Type - ", formatTypes[k])
+				volName := TestInputParamsUtil.GetVolumeNameWithTimeStamp("dockerVol")
+				_, err := TestUtil.InvokeCommand(vms[vmIndx], "docker volume create --driver=vmdk --name="+volName+" -o size="+volSizes[i]+" -o diskformat="+formatTypes[k])
+				if err != nil {
+					log.Fatalf("Failed to create a volume named: %s. Error - %v ", volName, err)
+				} else {
+					log.Println("\n successfully created a  volume -- ", volName)
+					dockerVolmRmvCmd = dockerVolmRmvCmd + " " + volName
+				}
+				log.Println("Verifying volume properties like size, disk-format and attached-to-vm fields at vm and esx for volume - ", volName)
+				volmPropertiesAdminCli := TestUtil.GetVolumePropertiesAdminCli(volName, esx)
+				expctdPropsAdmin := []string{"100.00MB", formatTypes[k], "detached"}
+				if !hasElement(expctdPropsAdmin, volmPropertiesAdminCli) {
+					log.Fatal("Volume properties on ESX fetched using admin cli does not matches the expected values")
+				}
+
+				if dockerCliCheck {
+					volmPropertiesDkrCli := TestUtil.GetVolumePropertiesDockerCli(volName, vms[vmIndx])
+					expctdPropsDkr := []string{"100MB", formatTypes[k], "<no value>"}
+					if !hasElement(expctdPropsDkr, volmPropertiesDkrCli) {
+						log.Fatal("Volume properties on admin cli do not matches the expected values")
```

Comment:
logging statement needs to be corrected ... while checking against docker cli outcome; logging should not be about "admin cli"